functions/helloworld/test/sample.system.storage.test.js

```diff
@@ -55,18 +54,20 @@ describe('system tests', () => {
     });
 
     // Wait for logs to become consistent
-    await promiseRetry(retry => {
+    for (let i = 0; i < 10; i++) {
```

Comment:
Exponential backoff for 10 cycles seems pretty arbitrary. Could we have a finally case in we keep slipping?

Reply:
I used 10 because that's what `promise-retry` defaulted to 🤷 anyhow, I decided this was changing too many things at once so I backed that part of the CL out and just focused on getting rid of request.

Reply:
Nit: if we're using `gaxios`, could we move the retry logic into that library (or perhaps a `gaxios-retry` library "shim") directly?